etc/DependencyInstaller.sh

```diff
@@ -188,7 +188,6 @@ _installUbuntuPackages() {
         libpcre3-dev \
         python3-dev \
         libreadline-dev \
-        libgtest-dev \
```

Comment:
Was there a library that should be uninstalled too?

Reply:
Not sure, I looked at when the dependency was introduced and it was only this line. See 9964e664f on #3062.

Reply:
See https://github.com/The-OpenROAD-Project/OpenROAD/discussions/3513#discussioncomment-6265721

Reply:
This is the only reference for `gtest` that I found. @QuantamHD can you confirm that the `gtest` you suggested uninstalling was just `libgtest-dev`?

Reply:
Yes. I think the other thing we might consider removing is the option of using the system Gtest in the cmake file. It seems like Gtest in general prefers to be included in your build rather that being a system dep.

Reply:
Sounds good. I will remove and also test this in a fresh Ubuntu 20.04 VM to make sure this does not break Harsh's or Peter's systems.